requirements.txt

```diff
@@ -2,7 +2,8 @@ pybullet >= 3.2.0
 numpy >= 1.21.5
 scipy >= 1.10.0
 spatialmath-python >= 1.0.0
-xacrodoc >= 0.4.0
+xacro >= 1.14.20
```

Comment:
I don't think `xacro` is actually required independently of `xacrodoc`, so it does not need to be listed in the dependencies.